air/src/air.rs

```diff
@@ -12,6 +14,8 @@ pub trait BaseAir<F>: Sync {
     fn preprocessed_trace(&self) -> Option<RowMajorMatrix<F>> {
         None
     }
+
+    fn columns(&self) -> Option<Vec<String>>;
```

Comment:
```suggestion
    /// If available, returns the names of the columns used in this AIR.
    /// If the result is `Some(names)`, `names.len() == air.width()` should always
    /// be true.
    fn columns(&self) -> Option<Vec<String>> {
        None
    }
```

Right? Then, we can move incrementally. I think then we also won't need any other changes in this Repo?

Reply:
Haha, yes, I realized the change after waking up and just changed it. That's why I deleted the part of my comment that talked about stark-backend in the orgininal PR. Will add the comment too.